project/plugins.sbt

```diff
@@ -1,4 +1,3 @@
-addSbtPlugin("com.dwijnand" % "sbt-travisci" % "1.2.0")
```

Comment:
This is slightly dangerous, since we're publishing based on `crossScalaVersions` in the build, and testing based on declarations in `.travis.yml`, and have no connection between the two.  But this is not new.